### PR TITLE
Route53 updater

### PR DESCRIPTION
**Pull request by hugoesthere**
Resolves: https://github.com/Originate/exosphere/issues/869
See: https://github.com/Originate/space-tweet/pull/43 for an example
 
Tested and works:
screen shot 2018-01-04 at 2 15 34 pm

### CODE REVIEW

File: resources/update-route53/main.go
Comment (charlierudolph) [inline resources/update-route53/main.go]: Lets use `log.Fataln` instead of `panic` as we don't need to have the stack trace printed

**Review by charlierudolph — APPROVED** [review]
Looks great to me. I added in #946 so we can add some documentation for how to use this. Please wait for a review from @alexdavid

File: resources/update-route53/main.go
Comment (charlierudolph) [inline resources/update-route53/main.go]: How about checking `if len(os.Args) != 3`?

File: terraform/aws/ecs-cluster/iam.tf
Comment (alexdavid) [inline terraform/aws/ecs-cluster/iam.tf]: I think we should restrict this to only allow changes on the internal hosted zone

File: resources/update-route53/main.go
Comment (alexdavid) [inline resources/update-route53/main.go]: does it make sense to run this as an external command or just make an http request here?

**Review by alexdavid — APPROVED** [review]

File: terraform/aws/ecs-cluster/iam.tf
Comment (hugoesthere) [reply terraform/aws/ecs-cluster/iam.tf]: How would we restrict it to only the internal hosted zone in aim roles?

**Pull request merged**
